access.ts

```diff
@@ -0,0 +1,50 @@
+import { permissionsList } from './schemas/fields'
+import { ListAccessArgs } from './types'
+
+export const isSignedIn = ({ session }: ListAccessArgs) => {
+  return !!session
+}
+const generatedPermissions = Object.fromEntries(
+  permissionsList.map((permission) => [
+    permission,
+    function ({ session }: ListAccessArgs) {
+      return !!session?.data.role?.[permission]
+    },
+  ]),
+)
+
+export const permissions = {
+  ...generatedPermissions,
+
+  //     you can add extra custom permission as well
+  //   canDoWhatever({ session }: ListAccessArgs) {
+  //     return session?.data.role?.canDoWhatever
+  //   },
+}
+
+export const rules = {
+  canManageSportClasses({ session }: ListAccessArgs) {
+    if (!isSignedIn({ session })) {
+      return false
+    }
+    return permissions.canManageSportClasses({ session }) ? true : false
+  },
+  canManageBookings({ session }: ListAccessArgs) {
+    if (!isSignedIn({ session })) {
+      return false
```

Comment:
Try to use expressions instead of booleans. 
In this case you could also just:
```
return isSignedIn({ session }) // yields to a boolean

```

Reply:
ok